apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -46,17 +47,24 @@ export class AITagView extends React.Component {
   }
 
   validateImage = async () => {
+    const FILE_LIMIT = 5000000; // 5MB limit from AWS Rekognition
+    const FILE_DIMENSION_LIMIT = 80; // 80px limit
     const imageId = get(this.props.entries.image.getValue(), 'sys.id');
-    if (!imageId) { return true; }
+    if (!imageId) { return ; }
 
     const file = await this.props.space.getAsset(imageId);
     const locale = this.props.locale;
     const contentType = get(file, `fields.file.${locale}.contentType`);
+    const details = get(file, `fields.file.${locale}.details`);
```

Comment:
Is `locale` guaranteed here?

Reply:
I'm not sure actually. I assumed all assets have a default locale. @jelz any gotchas I missed?

Reply:
Yes it's guaranteed, we use `sdk.locales.default` which is always present and has a valid code of the default locale